src/lib/ProtocolSummary/components/Rule.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { compose } from 'recompose';
+import RuleText from '@components/Query/Rules/PreviewText';
+import withDisplayOptions from '@components/Query/Rules/withDisplayOptions';
+
+const Rule = ({
+  type, options,
+}) => (
+  <RuleText type={type} options={options} />
+);
+
+Rule.propTypes = {
+  type: PropTypes.string.isRequired,
+  // eslint-disable-next-line react/forbid-prop-types
+  options: PropTypes.object.isRequired,
```

Comment:
Just a small thing with this, but if you are using the github co-pilot beta, it is quite good at auto-completing these.

Reply:
will look into it. :)